### DISCUSSION TITLE
PnP Search: Full Name search not returning results

### DISCUSSION THREAD

**Discussion by AlanTheta** [microsoft-search/pnp-modern-search#3100] [Q&A]
Hi all,

We have a People Search which doesn't return results when the user enters a full name. Think this is because its treating "FirstName LastName" as an exact single string. Is there a way we can edit the search query to effectively convert any spaces to a * wildcard, so that would be "FirstName*LastName" (which does return results)?

Thanks

Alan

**Comment by kasperbolarsen** [comment]
I am just using out of the box {searchTerms} in the Query Template. And it sure seems to work OK:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/148842e1-de4e-49bc-88bd-b0e97fb0bfda)
